library/std/src/io/error.rs

```diff
@@ -695,7 +695,7 @@ impl Error {
     #[stable(feature = "io_error_inner", since = "1.3.0")]
     #[must_use]
     #[inline]
-    pub fn get_ref(&self) -> Option<&(dyn error::Error + Send + Sync + 'static)> {
+    pub fn get_ref(&self) -> Option<&(dyn error::Error + Sync + 'static)> {
```

Comment:
`dyn Error + Send + Sync` has a large API surface that `dyn Error + Sync` doesn’t have.

Reply:
Also, in any case, changing these methods is a breaking change the impact of which I’m not sure has been tested anywhere yet.

Reply:
> that `dyn Error + Sync` doesn’t have. Like an [`is` method](https://doc.rust-lang.org/std/error/trait.Error.html#method.is-2) and a [`downcast_ref`](https://doc.rust-lang.org/std/error/trait.Error.html#method.downcast_ref-2) method.

[`dyn Error`  also implements](https://doc.rust-lang.org/std/error/trait.Error.html#impl-dyn+Error) `is` and `downcast_*` with exact the same signature.

Reply:
> [`dyn Error` also implements](https://doc.rust-lang.org/std/error/trait.Error.html#impl-dyn+Error) `is` and `downcast_*` with exact the same signature.

Oops, my bad. We cannot use `dyn Error`'s methods on `dyn Error + Sync`, though it seems odd to me.